package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "image-minimizer-webpack-plugin",
-  "version": "4.1.1",
+  "version": "4.1.2",
```

Comment:
@alexander-akait is the one bumping and publishing versions, so dont include that in this pr